examples/elevation/components/elevation/elevation.tsx

```diff
@@ -0,0 +1,61 @@
+import {useEffect} from 'react';
+import {
+  useElevationService,
+  useGoogleMap
+} from '@ubilabs/google-maps-react-hooks';
+
+const Elevation = () => {
+  const {map} = useGoogleMap();
+  // Get the elevator from the useElevationService hook
+  const elevator = useElevationService();
+
+  useEffect(() => {
+    if (!map) {
```

Comment:
Sorry, I just saw this in https://github.com/ubilabs/google-maps-react-hooks/pull/83. Could you add `!elevator` here the same way you did it for the max zoom service please?